### DISCUSSION TITLE
TypeError

### DISCUSSION THREAD

**Discussion by ronamadthis** [ourownstory/neural_prophet#1359] [Q&A - get help using NeuralProphet]
I am running NeuralProphet 3.2 in Python 3.7.6 environment. Until recently it was running fine. Then I decided to upgrade it to version 5.2. It gave error. Then I decided to downgrade it to version 3.2. It gave the same error. Then I decided to run in virtual environment. I got the same error as before. The TypeError is shown below:
TypeError                                 Traceback (most recent call last)
<ipython-input-1-fefc43db5619> in <module>
      1 import numpy as np
      2 from datetime import datetime
----> 3 from neuralprophet import NeuralProphet
      4 
      5 dfQ = pd.read_csv(r"C:\Users\martisr\Desktop\PyData\Code\Que_calls_6_22_23.csv", parse_dates = ['CAL_DATE'])

~\AppData\Roaming\Python\Python37\site-packages\neuralprophet\__init__.py in <module>
     23     warnings_log.addHandler(f_handler)
     24 
---> 25 from .forecaster import NeuralProphet
     26 from .utils import set_random_seed, set_log_level
     27 from .df_utils import split_df

~\AppData\Roaming\Python\Python37\site-packages\neuralprophet\forecaster.py in <module>
      9 from tqdm import tqdm
     10 
---> 11 from neuralprophet import configure
     12 from neuralprophet import time_net
     13 from neuralprophet import time_dataset

~\AppData\Roaming\Python\Python37\site-packages\neuralprophet\configure.py in <module>
     10 import types
     11 
---> 12 from neuralprophet import utils_torch, utils, df_utils
     13 
     14 log = logging.getLogger("NP.config")

~\AppData\Roaming\Python\Python37\site-packages\neuralprophet\utils_torch.py in <module>
      6 from torch_lr_finder import LRFinder
      7 
----> 8 from neuralprophet import utils
      9 
     10 log = logging.getLogger("NP.utils_torch")

~\AppData\Roaming\Python\Python37\site-packages\neuralprophet\utils.py in <module>
      6 import torch
      7 from collections import OrderedDict
----> 8 from neuralprophet import hdays as hdays_part2
      9 import holidays as pyholidays
     10 import warnings

~\AppData\Roaming\Python\Python37\site-packages\neuralprophet\hdays.py in <module>
    431 
    432 
--> 433 class TU(Turkey):
    434     pass
    435 

~\AppData\Roaming\Python\Python37\site-packages\holidays\registry.py in __init__(self, path, *args, **kwargs)
    180         if args:
    181             raise TypeError(
--> 182                 "This is a python-holidays entity loader class. "
    183                 "For entity inheritance purposes please import a class you "
    184                 "want to derive from directly: e.g., "

TypeError: This is a python-holidays entity loader class. For entity inheritance purposes please import a class you want to derive from directly: e.g., `from holidays.countries import Entity` or `from holidays.financial import Entity`.

Any help with regard to this is greatly appreciated. The error seems to be due to holidays module.

**Comment by leoniewgnr** [comment]
Hi, can you please install the latest version 0.6.2 and see if that error is still there?
